mlrose/neural.py

```diff
@@ -514,10 +516,10 @@ def fit(self, X, y=None, init_weights=None):
                 if init_weights is None:
                     init_weights = np.random.uniform(-1, 1, num_nodes)
 
-                current_weights, current_loss = random_hill_climb(
+                current_weights, current_loss, curve = random_hill_climb(
```

Comment:
The number of outputs of random_hill_climb() vary depending on whether curve is True or False, so you're going to need to use an if statement here.